plugins/logitech-hidpp/fu-logitech-hidpp-device.c

```diff
@@ -354,7 +355,14 @@ fu_logitech_hidpp_device_create_radio_child(FuLogitechHidPpDevice *self,
 	radio_version = g_strdup_printf("0x%.4x", build);
 	radio = fu_logitech_hidpp_radio_new(ctx, entity);
 	fu_device_set_physical_id(FU_DEVICE(radio), fu_device_get_physical_id(FU_DEVICE(self)));
-	fu_device_set_logical_id(FU_DEVICE(radio), priv->model_id);
+	/*
+	 * Use the parent logical id as well as the model id for the
+	 * logical id of the radio child device. This allows the radio
+	 * devices of two devices of the same type (same device type,
+	 * BLE mode) to coexist correctly.
+	 */
+	logical_id = g_strdup_printf("%s_%s", priv->model_id, fu_device_get_logical_id(FU_DEVICE(self)));
```

Comment:
I think the convention we have in other plugins is `PARENTVALUE-SOMETHINGELSE`, i.e. the other way around with a dash.

Reply:
> I moved the logical_id setup for BLE devices to fu_logitech_hidpp_device_setup

I think that works -- the device ID will change at that point, but it's before the device is added to the daemon and so I don't think that is matters. Thanks.